src/components/autocomplete.vue

```diff
@@ -46,10 +46,11 @@ export default {
       // get correct input from fallback or slot
       let refInput = _this.$refs.input
       if (_this.$slots.input) {
-        const refName = _this.$slots.input()[0].props.ref
-        const scopedInput = _this.$slots.input()[0].ref.i.ctx.$refs[refName]
+        const input = _this.$slots.input().find((el) => el.props.ref == 'input')
```

Comment:
For me, this line was failing, I put a comment inside the template:

```
<template #input="slotProps">
      <!-- Some comment  -->
      <input v-bind="slotProps" ref="input" />
  </template>
```

And got errors that props/ref was not defined. Could be solved by not putting comments in the template, but a better fix might be to check if the props/ref is defined:

`const input = _this.$slots.input().find((el) => el?.props?.ref && el.props.ref == 'input')`